### ISSUE TITLE
Can't restore postgres backup.

### ISSUE THREAD

**Issue by tfrum** [dbeaver/dbeaver#20650]
### Description

In 22 there's been a consistent bug where you cannot restore backups of a database with postgres.

This can be demonstrated and replicated by downloading any sample database and using the restore option.

pgAdmin4 successfully restores the same database to the same postgres server connection using the same local client.

Issue present with postgres13 and postgres14.

The specific error received is: 
Error executing process
Utility 'pg_restore' not found in client home '/lib/postgresql' (/lib/postgresql)

Which seems to indicate it isn't correctly looking in the postgresql/14 or postgresql/13 subdirectories where you point the local client.

Naturally, pg_restore is found in that client and other db management software, such as pgadmin, can use it successfully.

### DBeaver Version

22

### Operating System

Fedora 38 and 39

### Database and driver

Postgres14, postgres13

Default downloaded drivers for both in DB22 and DB23

**Comment by LonwoLonwo** [comment]
Hello @tfrum 

Did you try to add the path to pg_restore manually?

![2023-07-24 08_44_14-Restore](https://github.com/dbeaver/dbeaver/assets/45152336/fe538114-c244-427e-beb0-3ed1db867328)

![2023-07-24 08_44_56-Restore](https://github.com/dbeaver/dbeaver/assets/45152336/5892e87b-9517-4789-9c02-42634110fff7)

**Comment by tfrum** [comment]
It seems like you just told me to add the local client directory, which is necessarily something done already in order to get an error that references the local client directory. Did you thoroughly review what was in my report?

**Comment by tfrum** [comment]
Sorry, I should have been more specific. Yes, I did manually set the local client directory.

Under version 22 it throws this error whether or not you have the /lib/postgresql/13 or /lib/postgresql/14 subdirectory under Fedora 38/39. This error is not present in debian-like distributions and actually may have been fixed in 23--I jumped the gun a little reporting it here for both.

It should be noted that all that seems to be happening is the local client directory isn't being referenced correctly, and we're getting the parent postgresql directory.
